pkg/services/dashboardimport/api/api.go

```diff
@@ -73,8 +73,8 @@ func (api *ImportDashboardAPI) InterpolateDashboard(c *contextmodel.ReqContext)
 		return response.Error(http.StatusBadRequest, "bad request data", err)
 	}
 
-	if req.PluginId == "" {
-		return response.Error(http.StatusUnprocessableEntity, "pluginId must be set", nil)
+	if req.PluginId == "" && req.Dashboard == nil {
+		return response.Error(http.StatusUnprocessableEntity, "pluginId or dashboard must be set", nil)
```

Comment:
this should **and**, right?
```suggestion
		return response.Error(http.StatusUnprocessableEntity, "pluginId and dashboard must be set", nil)
```